Lib/test/test_cmd_line.py

```diff
@@ -459,6 +459,8 @@ def test_stdin_readline(self):
             "import sys; print(repr(sys.stdin.readline()))",
             b"'abc\\n'")
 
+    # TODO: RUSTPYTHON
+    @unittest.skipIf(sys.platform.startswith('win'), "TODO: RUSTPYTHON windows has \n troubles")
```

Comment:
I think other tests are marked `TODO: RUSTPYTHON, universal newlines`, but I like this description better, as it's a clearer statement of the problem.